### ISSUE TITLE
OGARCH

### ISSUE THREAD

**Issue by waynelapierre** [s-broda/ARCHModels.jl#85]
I am using your package with the book Financial Risk Forecasting. The only model in this book unavailable in your package is OGARCH. It would be great if you could add it to your great package. 

https://www.financialriskforecasting.com/code/RJulia3.html

**Comment by azev77** [comment]
I agree it would be great.
It’s in the feature wishlist:
https://github.com/s-broda/ARCHModels.jl/issues/60

**Comment by s-broda** [comment]
I agree, but am quite time constrained at the moment. If someone wants to tackle this, my preferred estimator would be the one from 

Broda, S.A. and Paolella, M.S. (2009). CHICAGO: A Fast and Accurate Method for Portfolio Risk Calculation”.
Journal of Financial Econometrics, Vol. 7(5), pp. 412–436. http://dx.doi.org/10.1093/jjfinec/nbp011

The FastICA algorithm that this requires appears to be implemented in https://github.com/JuliaStats/MultivariateStats.jl
